cogs/highlights.py

```diff
@@ -96,18 +121,21 @@ async def highlight(self, ctx: commands.Context, *, pattern: str) -> None:
             if await con.fetch_user_highlight_count(user_id := ctx.author.id) >= 10:
                 await ctx.send(":x: **Highlight limit exceeded.**")
                 return
-            await con.insert_highlight(user_id, pattern)
+            await con.insert_highlight(user_id, pattern, self._get_guild_id(server))
             await con.commit()
-        await ctx.send(f":white_check_mark: **Highlight pattern `{pattern.replace('`', '')}` added.**")
+        await ctx.send(f":white_check_mark: **Highlight pattern `{pattern.replace('`', '')}` added {self._server_text(server)}.**")
+
+    def _server_text(self, server, pre="", no_server="globally"):
```

Comment:
This method still needs type hints. In the code, it should be called `guild` instead of `server` unless it's visible to the user in which case `server` would be less confusing

Reply:
Oversight after refactoring  Using the terms `server`and `guild` interchangeable in code is a bit confusing, true.
But I would stick to the parameter name "server" for commands, since guild would confuse most users.